flink-libraries/flink-table/src/test/java/org/apache/flink/api/java/batch/table/UnionITCase.java

```diff
@@ -43,7 +43,7 @@ public UnionITCase(TestExecutionMode mode) {
 	}
 
 	@Test
-	public void testUnion() throws Exception {
+	public void testUnionAll() throws Exception {
```

Comment:
Actually, I would completely remove the whole Java `UnionITCase.java` file. It tests exactly the same methods as the Scala tests (no expression parsing involved).